subprojects/docs/src/docs/userguide/managing_transitive_dependencies.adoc

```diff
@@ -165,7 +165,66 @@ In the example, the versions of `gson` and `dom4j` are provided by the Spring Bo
 
 Gradle treats all entries in the `<dependencyManagement>` block of a BOM similar to <<#sec:dependency_constraints,Gradle's dependency constraints>>. This means that any version defined in the `<dependencyManagement>` block can impact the dependency resolution result. In order to qualify as a BOM, a `.pom` file needs to have `<packaging>pom</packaging>` set.
 
+However often BOMs are not only providing versions as recommendations, but also a way to override any over version found in the graph. You can enable this behavior by using the `enforcedPlatform` keyword, instead of `platform`, when importing the BOM:
+
+.Importing a BOM, making sure the versions it defines override any other version found in the graph
+====
+include::sample[dir="userguide/dependencyManagement/managingTransitiveDependencies/constraintsFromBOM/groovy",files="build.gradle[tags=dependency-on-bom]"]
+include::sample[dir="userguide/dependencyManagement/managingTransitiveDependencies/constraintsFromBOM/kotlin",files="build.gradle.kts[tags=dependency-on-bom]"]
+====
+
+[[sec:version_alignment]]
+== Dependency version alignment
+=== Handling inconsistent module versions
+Gradle supports aligning versions of modules which belong to the same "platform". It is often preferable, for example, that the API and implementation modules of a component are using the same version. However, because of the game of transitive dependency resolution, it is possible that different modules belonging to the same platform end up using different versions.
+For example, your project may depend on the `jackson-databind` and `vert.x` libraries, as illustrated below:
+
+.A dependency on the Jackson library
```

Comment:
In isolation, I'm not sure this title makes sense. The sample is more than what is said in the title, right?